### ISSUE TITLE
Use the same ODS code in the v0 and v1 demonstrators

### ISSUE THREAD

**Issue by jonnyry** [nhsconnect/gpconnect-demonstrator#152]
This is to allow the same PDS patient set can be used so that the registered practice ODS will work for both, and allow HTML and Structured records to be aligned.

Because the ODS code of the demonstrator needs to be aligned with PDS and SDS in multiple test environments, this needs to be done in a co-ordinated manner.

**Comment by jonnyry** [comment]
v1 is now using the new ODS code (A20047) and the new PDS patient set.  Committed to master on GitHub and released across DEV, INT,  OpenTest and the Internet Demonstrator.

v0 is using the old ODS code and old patient set in the master branch.  DEV and INT remain on this version due to ongoing consumer testing.
However v0 has been prepped with new ODS and patient set in a feature branch.  This has been released to OpenTest and the Internet Demonstrator.

We will reconvene in the new year to find a suitable time to update DEV and INT environments with the v0 demonstrator.

**Comment by jonnyry** [comment]
Could you make the following changes to the v0 (PDS aligned branch) and v1 demonstrators to align their Organization details for A20047 with the details held in SDS in OpenTest?

- v0 demonstrator
  a) Change the organisation name to "Dr Legg's surgery"
  b) Add a work address of NHS NPFIT, Test Data Manager, Princes Exchange, Leeds, LS1 4HY
  c) Add a telecom of "12345678" making it phone and work
  d) Remove the ODS site code identifier
  e) Change the ODS organization code identifier to A20047

- v1 demonstrator:
  a) Change the organisation name to "Dr Legg's surgery"
  b) Add a work address of NHS NPFIT, Test Data Manager, Princes Exchange, Leeds, LS1 4HY
  c) Remove the population of the "type" element altogether
  d) Add a telecom of "12345678" making it phone and work

current v0 demonstrator organization:

```
    {
      "fullUrl": "Organization/2",
      "resource": {
        "resourceType": "Organization",
        "id": "2",
        "meta": {
          "versionId": "1469448000000",
          "lastUpdated": "2016-07-25T12:00:00.000+00:00",
          "profile": [
            "http://fhir.nhs.net/StructureDefinition/gpconnect-organization-1"
          ]
        },
        "identifier": [
          {
            "system": "http://fhir.nhs.net/Id/ods-organization-code",
            "value": "R1A14"
          },
          {
            "system": "http://fhir.nhs.net/Id/ods-site-code",
            "value": "Z33433"
          }
        ],
        "name": "Test GP Care Trust"
      }
    }
```

current v1 demonstrator organization:

```
{
  "resourceType": "Organization",
  "id": "7",
  "meta": {
    "versionId": "1469448000000",
    "lastUpdated": "2016-07-25T12:00:00.000+00:00",
    "profile": [
      "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Organization-1"
    ]
  },
  "identifier": [
    {
      "system": "https://fhir.nhs.uk/Id/ods-organization-code",
      "value": "A20047"
    }
  ],
  "type": [
    {
      "coding": [
        {
          "system": "http://hl7.org/fhir/ValueSet/organization-type",
          "code": "dept",
          "display": "Hospital Department"
        }
      ]
    }
  ],
  "name": "HSCIC Test Practice",
  "telecom": [
    {
      "system": "phone",
      "use": "work"
    }
  ],
  "address": [
    {
      "use": "work",
      "type": "physical"
    }
  ]
}
```

A20047 details from SDS in OpenTest:

```
dn: uniqueIdentifier=A20047,ou=Organisations,o=nhs
nhsOrgTypeCode: PR
nhsParentOrgCode: 001
nhsJoinDate: 20071212
facsimileTelephoneNumber: 12345678
telephoneNumber: 12345678
mail: ANUBAHV@GDG.COM
nhsOrgType: GP Practice
nhsCountry: England
postalAddress: NHS NPFIT, TEST DATA MANAGER, PRINCES EXCHANGE, LEEDS
nhsPCTCode: 001
uniqueIdentifier: A20047
nhsIDCode: A20047
l: GWENT"YORKSHIRE AND THE HUMBER STRATEGIC HEALTH AUTHORITY"NORTH EAST CLUSTE
 R
nhsSHAcode: Q16
objectClass: organization
objectClass: top
objectClass: nhsGPPractice
nhsOrgOpenDate: 20071205
nhsDHSCcode: Y52
o: DR LEGG'S SURGERY
nhsAltOrgNames: A ORG
postalCode: LS1 4HY
nhsSyntheticIndicator: 0
```

**Comment by SimonFarrowNHS** [comment]
Fixed and committed in 1.n and fixed in code in 0.5 but not yet committed.

**Comment by SimonFarrowNHS** [comment]
Now applied to 0.5 as well
